pkgs/by-name/hu/huion-switcher/package.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  nix-update-script,
+  fetchFromGitHub,
+  rustPlatform,
+  udev,
+  pkg-config,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "huion-switcher";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "whot";
+    repo = pname;
```

referencing pname is discouraged 
```suggestion
    repo = "huion-switcher";
```

---

fixed